test/isolated/_resolvedata/js-ts-config/next.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  customConfigFileExt: 'js'
```

Comment:
Let's prefix this with an underscore so people know it's not a real config option.
```suggestion
  __test__ext: 'js'
```